authenticator-required-action/src/main/java/de/conciso/keycloak/authentication/required_action/RequiredActionAuthenticator.java

```diff
@@ -0,0 +1,44 @@
+package de.conciso.keycloak.authentication.required_action;
+
+import org.keycloak.authentication.AuthenticationFlowContext;
+import org.keycloak.authentication.Authenticator;
+import org.keycloak.models.KeycloakSession;
+import org.keycloak.models.RealmModel;
+import org.keycloak.models.UserModel;
+
+public class RequiredActionAuthenticator implements Authenticator {
+
+  @Override
+  public void authenticate(AuthenticationFlowContext context) {
+    context.getUser().addRequiredAction(
```

Comment:
It would be good if we could make this more robust to configuration errors, e.g. what if no authenticator config exists, or what if the key is not present in the config, or the configured required action does not exist.

Reply:
I will look into it.